### DISCUSSION TITLE
Does theming work for custom nodes?

### DISCUSSION THREAD

**Discussion by dacox** [facebook/lexical#4781] [Q&A]
I got the impression from reading https://lexical.dev/docs/getting-started/theming that I could simply attach css classes by doing `nodeType: className`.

However, on second reading, it seems that this only works for built in nodes. I'm having trouble finding confirmation.

Is this the case?

If not, how can I theme custom nodes (without explicitly attaching css classes in `createDOM`?

**Comment by moy2010** [comment]
No, theming does not apply to custom nodes. You need to style them separately.
